src/gmt_api.c

```diff
@@ -13527,7 +13527,7 @@ int GMT_Get_Default (void *V_API, const char *keyword, char *value) {
 	else if (!strncmp (keyword, "API_IMAGE_LAYOUT", 16U)) {	/* Report image/band layout */
 		gmt_M_memcpy (value, API->GMT->current.gdal_read_in.O.mem_layout, 4, char);
 	}
-	else if (!strncmp (keyword, "API_GIT_COMMIT", 14U))	/* The API version */
+	else if (!strncmp (keyword, "API_BIN_VERSION", 14U))	/* The API version */
```

Comment:
Not serious, but should have been **15U** instead of **14U**

Reply:
Updated, thanks